src/api/stakeholders.js

```diff
@@ -0,0 +1,189 @@
+import { handleResponse, handleError, getOrganizationId } from './apiUtils';
+import { session } from '../models/auth';
+const log = require('loglevel').getLogger('../api/stakeholders');
+
+const STAKEHOLDER_API = process.env.REACT_APP_STAKEHOLDER_API_ROOT;
+
+async function fetchJSON(query, options) {
+  return fetch(query, options).then(handleResponse).catch(handleError);
+}
+
+export default {
+  getStakeholders(id, { offset, rowsPerPage, orderBy, order, filter }) {
+    const orgId = id || getOrganizationId();
+    const filterObj = {
+      where: filter,
+      order: [`${orderBy} ${order}`],
```

Comment:
Right now, the `order` and `orderBy` params aren't being used by the stakeholder API, only by the Material table. As a user, if I were to sort, let's say alphabetically, I'd want to see all filtered rows sorted, not just the rows that are visible on the page. If sorting is only expected to order the rows that are returned from the paginated query (thus done in the client), then you can take out `order` from the `filterObj` (since the stakeholder API will always sort by ascending `org_name`. If sorting should be done on the backend, then updates would need to be made to the `stakeholder api` to utilize `options.order`. `handleSort()` in `Stakeholder/Table.js` should also setPage(0), similar to what is done when a filter is applied, to refresh the table.

Reply:
Thank you!  Good point.  Since we're unlikely to have a ton of stakeholders I think I'll drop the table pagination and querying to sort, so moving the sorting to the table makes sense.